src/main/java/org/fundacionjala/coding/Ruber/movies/Customer.java

```diff
@@ -0,0 +1,119 @@
+package org.fundacionjala.coding.Ruber.movies;
+
+import java.util.Enumeration;
+import java.util.Vector;
+
+/**
+ * Class Customer.
+ */
+class Customer {
+    private final String name;
+    private static Vector rentals = new Vector();
+    private static final int REGULAR_AMOUNT = 2;
+    private static final double CHILDRENS_AMOUNT = 1.5;
+    private static final double NORMAL_FINE = 1.5;
+    private static final double RELEASE_FINE = 3;
+    private static final int REGULAR_DAYS_RENTED = 2;
+    private static final int CHILDRENS_DAYS_RENTED = 3;
+
+    /**
+     * Constructor.
+     * @param name The customer's name
+     */
+    public Customer(String name) {
+        this.name = name;
+    }
+
+    /**
+     * This method add a new rental to the vector.
+     * @param arg The rental
+     */
+    public void addRental(Rental arg) {
+        rentals.addElement(arg);
+    }
+
+    /**
+     * This method returns the customer's name.
+     * @return customer's name
+     */
+    public String getName() {
+        return name;
+    }
+
+    /**
+     * This method calculates runs the entire vector to calculate.
+     * @return String with the result
+     */
+    public String statement() {
```

Comment:
you can split this method
currently is not following the Single responsibility principle
It is doing 3 things, it is calculating totalAmount, totalFrequentRenterPoints and it is printing too.